sdk/core/Azure.Core/src/Pipeline/Internal/RequestActivityPolicy.cs

```diff
@@ -54,7 +54,7 @@ public override void Process(HttpMessage message, ReadOnlyMemory<HttpPipelinePol
 
         private async ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPipelinePolicy> pipeline, bool async)
         {
-            using var scope = new DiagnosticScope("Azure.Core.Http.Request", s_diagnosticSource, message, s_activitySource, DiagnosticScope.ActivityKind.Client);
+            using var scope = new DiagnosticScope("Azure.Core.Http.Request", s_diagnosticSource, message, s_activitySource, DiagnosticScope.ActivityKind.Client, true);
```

Comment:
We wouldn't need to change this file if the constructor took a default value here.

Reply:
`RequestActivityPolicy` is special since it produces HTTP spans which should not be suppressed.

I added default value though (to suppress), but then it means that existing clients still need to set it (to not suppress).